app/api/events.py

```diff
@@ -104,6 +105,14 @@ class Meta:
                                  schema='EventInvoiceSchema',
                                  many=True,
                                  type_='event_invoice')
+    discount_codes = Relationship(attribute='discount_code',
```

Comment:
If we remove `discount_code_id` [here](https://github.com/fossasia/open-event-orga-server/blob/nextgen/app/models/event.py#L104), how will we describe the relationship between event and discount codes? see [line#105](https://github.com/fossasia/open-event-orga-server/blob/nextgen/app/models/event.py#L105)